backend/src/cache/client_manager.go

```diff
@@ -16,14 +16,16 @@ package main
 
 import (
 	"database/sql"
+	"encoding/json"
 	"fmt"
 	"log"
 	"time"
 
-	"encoding/json"
 	"github.com/cenkalti/backoff"
 	"github.com/golang/glog"
 	"github.com/jinzhu/gorm"
+	api_client "github.com/kubeflow/pipelines/backend/src/apiserver/client"
+	api_storage "github.com/kubeflow/pipelines/backend/src/apiserver/storage"
```

Comment:
maybe apiserver_client and apiserver_storage ?

Reply:
yepp, will rename it.